src/common/users.ts

```diff
@@ -102,12 +104,14 @@ export const getUserReadingRank = async (
       `count(*) filter(where "timestamp" at time zone '${timezone}' >= date_trunc('day', ${now}))`,
       'today',
     )
+    .addSelect('MAX(timestamp)', 'lastReadTime')
```

Comment:
Thanks to this we no longer need the previous select `today`. We used it to calculate the `readToday` prop. But now we can use the `lastReadTime` for it. Just remember to convert to the right time zone when comparing days.

Reply:
Thank you for noting this! I have now applied the changes 🚀